src/main/resources/config-schema-storage-driver-kafka.yml

```diff
@@ -4,6 +4,8 @@ load:
     timeoutMillis: int
 storage:
   driver:
+    record:
```

Comment:
I don't think we need the configuration parameter. Just read in the batch mode always

Reply:
Ok. Removed